Update repository table

Ein-Tim opened the pull request:
This PR updates the repositories table to reflect the latest created repositories (~https://github.com/corona-warn-app/cwa-quick-test-frontend, https://github.com/corona-warn-app/cwa-quick-test-backend~, https://github.com/corona-warn-app/cwa-event-landingpage, https://github.com/corona-warn-app/cwa-event-qr-code & https://github.com/corona-warn-app/cwa-quicktest-onboarding)

This PR is in draft mode since there isn't an official description for two of the repos yet.

TODO:

- [ ] Add description for https://github.com/corona-warn-app/cwa-quick-test-backend - **Will be added in #625**
- [ ] Add description for https://github.com/corona-warn-app/cwa-quick-test-frontend - **Will be added in #625**
- [x] Add some missing .

 Internal Tracking ID: [EXPOSUREAPP-7369](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7369) repo tables
 Internal Tracking ID: [EXPOSUREAPP-6382](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6382) more docs

Ein-Tim committed:
Update repository table

This commit updates the repositories table to reflect the latest created repositories.

Ein-Tim commented:
@dsarkar Could you ask for a description that I can use? 
I already opened https://github.com/corona-warn-app/cwa-quick-test-frontend/issues/35 & https://github.com/corona-warn-app/cwa-quick-test-backend/issues/14, but so far nothing has changed.

Thank you!

dsarkar commented:
@Ein-Tim This issue has high priority https://github.com/corona-warn-app/cwa-wishlist/issues/449, so I will forward the question to that ticket.

dsarkar commented:
see also Internal Tracking ID: [EXPOSUREAPP-6382](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6382)

Ein-Tim commented:
Thank you!

Ein-Tim committed:
Add cwa-event-landingpage

This commit adds another new repo to the list of repositories.

Ein-Tim commented:
[`1db6069` (#570)](https://github.com/corona-warn-app/cwa-documentation/pull/570/commits/1db6069a041d5ea044ad0d3f36e814a570df9bde) added https://github.com/corona-warn-app/cwa-event-landingpage

Ein-Tim committed:
Add cwa-event-qr-code

This commit adds another new repo to the list of repositories.

Ein-Tim commented:
https://github.com/corona-warn-app/cwa-documentation/pull/570/commits/82aadebe56051565db00cda09ab77ad7f33604f1 added https://github.com/corona-warn-app/cwa-event-qr-code

Ein-Tim commented:
Check is failing because of markdown errors in https://github.com/corona-warn-app/cwa-documentation/blob/master/event_registration.md. 
I'll provide a PR to fix them.

Ein-Tim committed:
Add cwa-quicktest-onboarding

This commit adds another new repo to the list of repositories.

Ein-Tim commented:
https://github.com/corona-warn-app/cwa-documentation/pull/570/commits/66cda08f9eebadfe970d3a9f2370ae86e956cf6d added https://github.com/corona-warn-app/cwa-quicktest-onboarding

Ein-Tim committed:
Add missing . and improve wording

Ein-Tim commented:
@dsarkar Could you ask for an update reg. the description texts here, please? 
For people with insight it should be a task done in 2 minutes, just writing one sentences what the `quick-test-frontend` and what the `quick-test-backend` repos do.

Thank you!

dsarkar commented:
@Ein-Tim requested info in Internal Tracking ID: [EXPOSUREAPP-7369](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7369)

MikeMcC399 commented:
@Ein-Tim 
Perhaps you could separately add the repository
https://github.com/corona-warn-app/cwa-quicktest-onboarding ?

See also https://github.com/corona-warn-app/cwa-documentation/issues/595#issuecomment-849055519

Ein-Tim commented:
@MikeMcC399 I think I'll make this PR merge ready and just leave out https://github.com/corona-warn-app/cwa-quick-test-frontend & https://github.com/corona-warn-app/cwa-quick-test-backend.

Then we can update the repo list and I'll just open another PR which waits for feedback on the texts...

MikeMcC399 commented on README.md:
```diff
@@ -88,20 +88,25 @@ For an easier understanding of the used acronyms and special terms in our docume
+| [cwa-quick-test-backend]   | waiting for description                                                           |
+| [cwa-event-landingpage]    | Landing page for CWA which is opened if the user does not have the app installed. |
+| [cwa-event-qr-code]        | Utility to generate QR codes for Event Registration.                              |
+| [cwa-quicktest-onboarding] | Informations about onboarding procedure for partners of the Corona-Warn-App.      |
```
Informations => Information

https://dictionary.cambridge.org/dictionary/english/information
"Information is an uncountable noun meaning ‘facts about someone or something’. It is often followed by about: …"

Maybe include also "rapid test" in the description, for instance:

"Documentation for integration of rapid test partners."

which is also a bit shorter.

Ein-Tim replied:
Done in https://github.com/corona-warn-app/cwa-documentation/pull/570/commits/d7ea0cc8a0f1fe8662fd9d1c95c2377374202fc7

MikeMcC399 reviewed (changes requested):
There is a grammatical error in the description for quicktest, and I made an alternative suggestion for the description.

Ein-Tim committed:
Remove repos without description

This commit removes the repos without a description and changes the description of the quicktest-onboarding repo.

Ein-Tim commented:
This PR is now ready to be merged.
https://github.com/corona-warn-app/cwa-quick-test-frontend & https://github.com/corona-warn-app/cwa-quick-test-backend will be added in another PR.

heinezen commented:
Looks fine!
